clusterloader2/pkg/measurement/common/slos/windows_node_resource_usage.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package slos
+
+import (
+	"math"
+	"time"
+
+	"github.com/prometheus/common/model"
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	windowsResourceUsagePrometheusMeasurementName = "WindowsResourceUsagePrometheus"
+	// get top 10 non-system processes with highest cpu usage within 1min query window size
+	cpuUsageQueryTop10 = `topk(10, sum by (process) (rate(wmi_process_cpu_time_total{process!~"Idle|Total|System"}[1m]) / on(job) group_left wmi_cs_logical_processors) * 100)`
+)
+
+type windowsResourceUsageMetricMeasurement struct {
+	resourceConstraints map[string]*measurementutil.ResourceConstraint
+}
+
+type windowsResourceUsageGatherer struct{}
+
+func (w *windowsResourceUsageGatherer) IsEnabled(config *measurement.MeasurementConfig) bool {
+	return true
+}
+
+func (w *windowsResourceUsageGatherer) String() string {
+	return windowsResourceUsagePrometheusMeasurementName
+}
+
+func init() {
+	create := func() measurement.Measurement { return createPrometheusMeasurement(&windowsResourceUsageGatherer{}) }
+	if err := measurement.Register(windowsResourceUsagePrometheusMeasurementName, create); err != nil {
+		klog.Fatalf("Cannot register %s: %v", windowsResourceUsagePrometheusMeasurementName, err)
+	}
+}
+
+func convertToPerfData(samples []*model.Sample) *measurementutil.PerfData {
+	perfData := &measurementutil.PerfData{Version: currentAPICallMetricsVersion}
```

Comment:
Let's not mix constants across different measurements, this measurement has nothing to do with API Call Latency one.

Reply:
fixed